.github/workflows/build.yml

```diff
@@ -53,6 +53,9 @@ jobs:
       - name: Check out the code
         uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
 
+      - name: Add repository directory to the git global config as a safe directory
+        run: git config --global --add safe.directory /__w/parca-agent/parca-agent
```

Comment:
This one should not be necessary, I've only seen affecting job containers so far

Reply:
The snap workflow can use one tho

Reply:
Thanks!!